libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/NavigationMapRoute.java

```diff
@@ -483,11 +477,43 @@ private void updateProgressChangeListener() {
     }
   }
 
+  /**
+   * Updates the size of the bounding box used to determine which route line was clicked. Upon
+   * a map click an attempt will be made to find a route line intersecting the bounding box. This
+   * is used to select alternative route lines on the map.
+   *
+   * Passing large integers values through this method means that a user doesn't
+   * have to be very accurate with tapping on an actual route line because the invisible
+   * box is large and thus, there's a good chance that a portion of a route line will run
+   * through the query box's area. A smaller rectangle will decrease the chances.
+   */
+  public void updateClickDistancePadding(int newRouteClickRectFPadding) {
+    routeClickPadding = newRouteClickRectFPadding;
+  }
+
+  @Nullable
```

Comment:
This can be removed :)

Reply:
With Langston's example activity the padding can be changed dynamically in order to see the effect.  It seems useful to me that a developer might want to change the padding at runtime, perhaps for different zoom levels.

Reply:
The padding we are adding is in pixel values, it's unrelated to a zoom level, and I think it should remain constant across a nav session to deliver a consistent experience. The on-screen area which we search in when the map is touched should remain the same. This value is useful to improve the experience across different screen dimensions and pixel densities.

This uncovers another thing we missed - the default padding value should be scaled by pixel density, it cannot remain constant. Could you address that?

Reply:
Also, this particular comment was about unnecessary `@Nullable` annotation.

Reply:
Also, we can always expose mutability in the future if there's a strong request.

Reply:
Ok, this unfortunately hobbles the example activity a bit since it demonstrates the sizing of the click area.  I'll make the adjustments you asked for.